audit/projects/k8s-infra-public-pii/services/bigquery/bigquery.datasets.k8s_infra_artifacts_gcslogs.access.json

```diff
@@ -3,6 +3,10 @@
     "role": "WRITER",
     "specialGroup": "projectWriters"
   },
+  {
+    "role": "WRITER",
+    "userByEmail": "service-226195303281@gcp-sa-bigquerydatatransfer.iam.gserviceaccount.com"
```

Comment:
I thought the service account `bq-data-transter` was going to be used instead of the agent service when I triggered a manual transfer. Am I wrong ?